py-rse/git-cmdline.Rmd

```diff
@@ -1175,19 +1188,35 @@ This file can contain filenames like `thesis.pdf`
 or [wildcard][wildcard] patterns like `*.dat`.
 Each must be on a line of its own,
 and Git will ignore anything that matches any of these lines.
+For example,
+the `.gitignore` file for the Zipf's Law project contains:
+
+```text
+.Rhistory
+.Rproj.user
```

Comment:
This volume doesn't require the installation of RStudio, so it might be a bit confusing to add `.Rhistory` and `.Rproj.user` to the `.gitignore` file?

Reply:
:+1: